utils.groovy

```diff
@@ -199,4 +199,28 @@ def get_push_trigger() {
     ]
 }
 
+def change_metal_artifacts_list_order(artifacts) {
+    if (artifacts.contains('live')) {
```

Comment:
should we check inside here to make sure both `metal` and `metal4k` exist?

Reply:
Um, not sure. I guess if we have `live` we must have these other two? Or is it not always the case?

Reply:
IIUC if you have `live` you need the others. @jlebon would know if `metal4k` is optional, but at least `metal` I know is required.

Reply:
We can leave it like it is in the code right now and do any fixups later.